src/api/public/apidocs-new/components/parameters/user_login.yaml

```
@@ -0,0 +1,7 @@
+in: path
+name: user_login
+schema:
+  type: string
+required: true
+description: User login.
+example: Adm
```

:nitpick: In order to have a working example, at least for our development environment. 

```suggestion
example: Admin
```